CHANGELOG.md

```diff
@@ -12,7 +12,29 @@
                     info: "Name of the assigned tag"
         end
     ```
-  - Support generation of markdown documentation for the nested block fields.
+  - Add support for nested parameters on an endpoint.
+    ```ruby
```

Comment:
just curious what does 
```
```ruby
```
do here?

Reply:
It helps with syntax highlighting in code blocks. https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting